helix-core/src/main/java/org/apache/helix/controller/rebalancer/strategy/crushMapping/CardDealingAdjustmentAlgorithmV2.java

```diff
@@ -55,6 +55,26 @@ public CardDealingAdjustmentAlgorithmV2(Topology topology, int replica, Mode mod
     }
   }
 
+  public Map<Node, Node> getInstanceFaultZone() {
+    return _instanceFaultZone;
+  }
+
+  public Map<Node, Long> getInstanceWeight() {
+    return _instanceWeight;
+  }
+
+  public long getTotalWeight() {
+    return _totalWeight;
+  }
+
+  public Map<Node, Long> getFaultZoneWeight() {
```

Comment:
Add methods into a class just for testing is not quite recommended.  If you have to do that, setting fields to protected are a bit better than adding more codes here?

Reply:
Making it `protected` field won't make it better. `protected` is not designed for unit testing purpose but for inheritance and it exposes the internal field directly. I could make the methods package private so these methods won't be misused publicly